From 7d3f9e19079ea0f6373c7d486e5ecd062a7a3fa8 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 13 Aug 2018 11:54:27 -0700
Subject: [PATCH] Revert "Makefile: force sync after build in case reboot loses

 executable"

This reverts commit ae6322b1baea56a589207c96e358daae8edd0a8f.

Not needed for cross-builds

Upstream-Status: Inappropriate [Cross compile specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 093d9c8..f876e0f 100644
--- a/Makefile
+++ b/Makefile
@@ -371,7 +371,6 @@ OBJS += $(CONFIG_OBJS)
 stress-ng: $(OBJS)
 	@echo "LD $@"
 	@$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -o $@
-	@sync
 
 makeconfig:
 	@if [ ! -e config ]; then \
